R/process_tests_sc_client_lookup.R

```diff
@@ -0,0 +1,64 @@
+#' Social care client lookup tests
+#'
+#' @description This script takes the processed social care client lookup and
+#' produces a test comparison with the previous data. This is written to
+#' disk as a CSV.
+#'
+#' @inherit process_tests_acute
+#'
+#' @export
+process_tests_sc_client_lookup <- function(data, year) {
+  old_data <- get_existing_data_for_tests(data)
+
+  comparison <- produce_test_comparison(
+    old_data = produce_tests_sc_client_lookup(old_data),
+    new_data = produce_tests_sc_client_lookup(data)
+  )
+
+  comparison %>%
+    write_tests_xlsx(sheet_name = "sc_client", year)
+
+  return(comparison)
+}
+
+
+#' Social care Client lookup Tests
+#'
+#' @description Produce the test for the social care Client all episodes
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path()])
+#' @param max_min_vars variables used when selecting 'min-max' from [calculate_measures()]
+#' @return a dataframe with a count of each flag.
+#'
+#' @family social care test functions
```

Comment:
If using this 'family' (I think it's a new one) you should also edit `_pkgdown.yml` to make sure they appear there.